userguide/config.toml

```diff
@@ -42,11 +42,6 @@ anchor = "smart"
     url = "https://example.docsy.dev"
     post = '<sup><i class="pl-1 fas fa-external-link-alt fa-xs" aria-hidden="true"></i></sup>'
 
-[services]
-[services.googleAnalytics]
-# Comment out the next line to disable GA tracking. Also disables the feature described in [params.ui.feedback].
-id = "UA-00000000-0"
```

Comment:
@LisaFC @emckean @geriom - I suspect that the all-zeros ID is bogus, and that it was used only for the side-effect of having the feedback form appear on pages. So I'm removing this for now, and we can track progress via:

- #1097
- #1099

Reply:
Update: I moved the ID to earlier in the file.